onnxruntime/python/tools/transformers/benchmark.py

```diff
@@ -366,13 +367,13 @@ def run_pytorch(
                     device=device,
                 )
                 try:
-                    inference = torch.jit.trace(model, input_ids) if torchscript else model
+                    inference = torch.jit.trace(model, input_ids) if torchscript else torch.compile(model) if torch2 else model
```

Comment:
Does stable version of pytorch has torch.compile API? You might check whether this script could run in pytorch 1.*.

Reply:
Added a check for PyTorch version